### PR TITLE
Added the timer view and fixed the tab icons

### PR DESCRIPTION
**Pull request by jdkouris**
# Description

Created the timer screen so users can set how much time they want to spend on tasks. Made the tab bar icons look like a 'home' and 'timer'

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

### CODE REVIEW

**Comment by jdkouris** [comment]
Hey Vici. This is a large PR. Right now the TimerVC file contains the TimerVC class + the Countdown class. We can move that second class to a new file if it makes more sense, but I didn't want to create a new project file and mess with the base project file while you were working. Let me know what you think

**Comment by vshaweddy** [comment]
> Hey Vici. This is a large PR. Right now the TimerVC file contains the TimerVC class + the Countdown class. We can move that second class to a new file if it makes more sense, but I didn't want to create a new project file and mess with the base project file while you were working. Let me know what you think

You can create another file for the Countdown. I will pull the latest from you first before I do my PR tonight.

File: Get2It/Get2It/Screens/TimerVC.swift [hunk 1]
```diff
@@ -8,23 +8,251 @@
...
     }
-    */
+    
+    func countdownDidFinish() {
+        updateViews()
+        showAlert()
+    }
+}
 
+class Countdown {
```
Comment (vshaweddy) [inline Get2It/Get2It/Screens/TimerVC.swift]: I think it's better to have this class in a separate file like you suggested.

Comment (jdkouris) [reply Get2It/Get2It/Screens/TimerVC.swift]: Just created a new file

Comment (vshaweddy) [reply Get2It/Get2It/Screens/TimerVC.swift]: thank you

**Pull request merged**